zerver/tests/test_upload.py

```diff
@@ -428,6 +429,10 @@ def test_valid_avatars(self):
             zerver.lib.upload.upload_backend.ensure_medium_avatar_image(user_profile.email)
             self.assertTrue(os.path.exists(medium_avatar_disk_path))
 
+            # Verify whether the avatar_version gets incremented with every new upload
+            self.assertEqual(user_profile.avatar_version, version)
+            version += 1
+
     def test_invalid_avatars(self):
         # type: () -> None
         """
```

Comment:
@isht3 I think it will be good to have a negative test as well. Test that the avatar version is not incremented incase of invalid avatars. Thanks!! :)